woocommerce/woocommerce-android
# Add toolbar for Order list and Order detail

Part of #10570 

### Description

This PR introduces individual toolbars for Order list and Order detail screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order list and Order detail screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="432" alt="Screenshot 2024-02-12 at 5 35 57 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/456bf62a-1de2-4c75-b00c-05d8f75465d9">
<img width="431" alt="Screenshot 2024-02-12 at 5 36 09 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/351b5bef-65e2-47e2-b9bc-0cc7d8111a58">
